Source/Tx.Windows/PerfCounters/PerfCounterRealTimeProbe.cs

```diff
@@ -55,7 +55,10 @@ public void OnTimer(object state)
 
         public override void Dispose()
         {
-            _timer.Dispose();
+            if (_timer != null)
+            {
+                _timer.Dispose();
+            }
```

Comment:
Please add an unit test for the change (similar to [this one](https://github.com/Microsoft/Tx/blob/master/Test/UnitTests/PerfCounterTest.cs#L154)), that would have also helped you to identify the flow with the proposed changes.